apiutils/task.go

```diff
@@ -0,0 +1,21 @@
+package apiutils
```

Comment:
Yeah, I'm with @stevvooe on this one.

Also, it's not really testing equality - it's testing some kind of equality that disregards some arbitrary fields.

If someone imports this package and uses `TasksSpecEqual` expecting it to actually do an end to end comparison, it's going to lead to unexpected behavior.

Today the dispatcher and agent versions are the same, but they're not bound to be. I'm not sure why the dispatcher would care about any change but the spec (while the helper function still compares everything but meta and status).

Since it's a 4 lines function, I'd prefer for it to be copied as it's not meant to be generic